pages/login.tsx

```diff
@@ -98,12 +103,12 @@ export default function Login() {
         $setLoaded(true)
         $setMessage('Logged in!')
```

Comment:
There should be a data refetch triggered on the `useLogin` context here.

The login button is currently updating post-redirect due to Next.js useRouter triggering a data refetch as mentioned above, but the user button should update (or start updating) as soon as we know the user has successfully logged in.

Reply:
I guess so -- this is a weird one to me because of how Magic Link works (i.e. This is the original `/login` page that sits around while the user goes to their email and then there's another page with a separate state later.) I kinda considered this original page not really a use-case but I'll add the data here.